### DISCUSSION TITLE
USB Midi Host?

### DISCUSSION THREAD

**Discussion by deepbluemind74** [sekigon-gonnoc/Pico-PIO-USB#10] [Q&A]
Would it be possible to build a USB Midi Host with this setup?

**Comment by Spartacus1066** [comment]
I **### second** the suggestion! **Thanks for your work!**

**Comment by rppicomidi** [comment]
@deepbluemind74 @Spartacus1066 Yes. See [pico-usb-midi-filter](https://github.com/rppicomidi/pico-usb-midi-filter) for an example.
